src/components/structures/MatrixChat.js

```diff
@@ -1427,5 +1474,7 @@ module.exports = React.createClass({
                 />
             );
         }
+
+        throw new Error(`Unknown view ${this.state.view}`);
```

Comment:
I'm not sure if anything is gained by throwing an error here. Could just log it?